core/dictgen/src/rootcling_impl.cxx

```diff
@@ -2775,6 +2776,12 @@ int GenerateFullDict(std::ostream &dictStream,
    // Loop to write all the ClassCode
    if (!gOptIgnoreExistingDict) {
       for (auto const &selClass : scan.fSelectedClasses) {
+         // The "isGenreflex" parameter allows the distinction between
+         // genreflex and rootcling only for the treatment of collections which
+         // are data members. To preserve the behaviour of the original
+         // genreflex and rootcling tools, if the selection is performed with
+         // genreflex, the type of the collections which are data members are
```

Comment:
How can a type of a collection be a data member? Do you mean "data members with collection type trigger selection of the element type but not of the collection type"?

Reply:
Maybe "data members with collection type trigger neither the selection of the element type nor of the collection type".  Actually since we never 'trigger the selection of the element type', simply "data members with collection type does not trigger the selection of the collection type"

Reply:
Suggestion now implemented.